nomad/jitsi_packs/packs/jitsi_autoscaler/templates/jitsi_autoscaler.nomad.tpl

```diff
@@ -33,29 +33,55 @@ job [[ template "job_name" . ]] {
     }
 
     network {
+      mode = "bridge"
       port "http" {
         to = 8080
       }
-      port "metrics" {
-        to = 8081
+      port "expose" {
```

Comment:
is there a reason that we're calling this expose2 instead of metrics? it seems to make things less clear in the code

Reply:
Great question, this is a workaround for specifying multiple expose paths on the same port in the nomad service.sidecar.proxy

https://github.com/hashicorp/nomad/issues/10841

Reply:
ew.